scipy/stats/_distn_infrastructure.py

```diff
@@ -3106,6 +3110,21 @@ def _pmf(self, k, *args):
     def _logpmf(self, k, *args):
         return log(self._pmf(k, *args))
 
+    def _logpxf(self, k, *args):
+        # continuous distributions have PDF, discrete have PMF, but sometimes
+        # the distinction doesn't matter. This lets us use `_logpxf` for both
+        # discrete and continuous distributions.
+        return self._logpmf(k, *args)
+
+    def _unpack_loc_scale(self, theta):
```

Comment:
This is copy-pasted from `rv_continuous` - just changed so that `scale` is always `1`.